math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -667,6 +667,27 @@ void swap(RVec<T> &lhs, RVec<T> &rhs)
    lhs.swap(rhs);
 }
 
+/// Return indices that sort the vector
+template <typename T>
+RVec<size_t> Argsort(const RVec<T> &v)
+{
+   using size_type = typename std::vector<T>::size_type;
+   RVec<size_type> i(v.size());
```

Comment:
Return is `RVec<size_t>` while this is `RVec<size_type>`. Maybe it's better to keep it consistent by using the same type in both places.

Reply:
ah right, forgot the `size_t` here from my proof of concept implementation